test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- According to the project requirements, this file should be ignored by git, hence shouldn't be visible in your repo. Please make sure that the `.gitignore` ignores this file 🙏. You might find this [article](https://www.freecodecamp.org/news/gitignore-file-how-to-ignore-files-and-folders-in-git/) helpful in knowing more about this 👍